src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(filePath))) {
```

Comment:
Ensure that you use try-with-resources with classes that implement AutoCloseable. Here, you've done it correctly by using BufferedReader within a try-with-resources statement.